routes/userRoute.js

```diff
@@ -15,6 +15,43 @@ const createToken = (id) => {
    });
 }
 
+router.get('/', async (req, res) => {
+    const token = req.cookies.jwt;
+    if (!token) return res.status(400).send('Cookie was not found.')
+    const decoded = jwt.verify(token, process.env.TokenSecret);
+    var userId = decoded.id;
+    const user = await User.findOne({ _id: userId });
+    if (!user) return res.status(400).send('User was not found.');
+
+    console.log('Successfully found User.');
+    res.status(200).send(user);
+});
+
+router.put('/', async (req, res) => {
+    const token = req.cookies.jwt;
+    if (!token) return res.status(400).send('Cookie was not found.')
+    const decoded = jwt.verify(token, process.env.TokenSecret);
+    var userId = decoded.id;
+    const user = await User.findOne({ _id: userId });
+    if (!user) return res.status(400).send('User was not found.');
+
+    const update = {
+        ...(req.body.firstName && { firstName: req.body.firstName }),
+        ...(req.body.lastName && { lastName: req.body.lastName }),
+        ...(req.body.email && { email: req.body.email }),
+        ...(req.body.password && { password: req.body.password })
```

Comment:
I think we need to hash the password before storing similar to /register endpoint.